config/stack/manifests/app.yaml

```diff
@@ -1,5 +1,5 @@
 # Human readable title of application.
-title: Crossplane Rook
+title: Crossplane Rook Provider
 
 overviewShort: The Rook Crossplane Stack enabling resource management for Rook.
```

Comment:
want to make other changes to this file to go from `Rook Crossplane Stack` -> `Rook Crossplane Provider`?